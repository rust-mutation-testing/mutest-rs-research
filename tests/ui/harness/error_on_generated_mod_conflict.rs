//@ build: fail
//@ stderr

mod mutest_generated {}
