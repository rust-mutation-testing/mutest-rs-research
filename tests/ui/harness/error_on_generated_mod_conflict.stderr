error: mutest-injected module conflicts with existing item `mutest_generated`
  |
  = note: mutest injects a module into the crate root with the reserved name `mutest_generated`

error: aborting due to 1 previous error

